docs/components/modeler/web-modeler/modeling-dmn.md

```diff
@@ -0,0 +1,49 @@
+---
+id: modeling-dmn
+title: Modeling DMN
+description: Let's create a DMN diagram.
+---
+
+## Create new DMN decision requirement diagram
```

Comment:
@SebastianStamm then let's move this one to `launch-cloud-modeler` and mention there that you can create DMN and BPMN.

Reply:
So essentially after this part is moved, we should remove the page with all the screenshots.